docs/source/tutorials/datasets.rst

```diff
@@ -52,15 +52,15 @@ function. You can see a list of all supported datasets :ref:`here<datasets>`.
     tune run full_finetune_single_device --config llama3/8B_full_single_device \
     dataset=torchtune.datasets.alpaca_dataset
 
-Hugging Face datasets
----------------------
+Hugging Face and Local datasets
```

Comment:
Local is already covered in a different section, let's remove it from here to avoid confusion?